homeassistant/components/netatmo/camera.py

```diff
@@ -313,10 +319,21 @@ async def _service_set_persons_home(self, **kwargs: Any) -> None:
         """Service to change current home schedule."""
         persons = kwargs.get(ATTR_PERSONS, {})
         person_ids = []
+        person_id_errors = []
+
         for person in persons:
-            for pid, data in self._data.persons.items():
+            person_id = None
+            for pid, data in self._data.persons[self._home_id].items():
                 if data.get("pseudo") == person:
                     person_ids.append(pid)
+                    person_id = pid
```

Comment:
Can there be more than one person id per person? Otherwise we can break here when finding a match.

Reply:
Thanks for the feedback. We can break early indeed.